features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -708,6 +708,9 @@
 
         <PublishPasswordGrantLogin>{{analytics.publish_password_grant_logins}}</PublishPasswordGrantLogin>
 
+        <DCRM>
+            <ApplicationRolePermissionRequiredToView>{{oauth.dcrm.application_role_permission_required_to_view}}</ApplicationRolePermissionRequiredToView>
```

Comment:
Better if we put some description about the property

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3113/commits/8218abc90dd0611e61183be14012b9fb7b9ceb3b